spec/models/author_spec.rb

```diff
@@ -1,5 +1,16 @@
 require 'rails_helper'
 
 RSpec.describe Author, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  context 'method checking' do
+    it 'if data stores' do
+      author = Author.new(id: 1, name: 'abc', email: 'abc@gmail.com')
+      expect(author).to be true
```

Comment:
I don't see the purpose of this test case. new method creates an object. you should at least validate the object or save this to database right?.